docs/t-sql/statements/alter-database-transact-sql.md

```diff
@@ -976,8 +976,7 @@ Provides the syntax and related information for changing the attributes of a dat
 ```syntaxsql
 ALTER DATABASE { database_name | CURRENT }
 {
-  MODIFY NAME = new_database_name
-| MODIFY ( <edition_option> [, ... n] )
+  MODIFY ( <edition_option> [, ... n] )

```

Comment:
```suggestion
  MODIFY NAME = new_database_name
| MODIFY ( <edition_option> [, ... n] )
```
Reverting this change as it does apply to dedicated SQL pools formerly SQL DW created via the Azure portal, and we clarify this later in the arguments section.

Reply:
#sign-off